### ISSUE TITLE
Add the nonce in the ChannelClosed event

### ISSUE THREAD

**Issue by loredanacirstea** [raiden-network/raiden-contracts#249]
as @hackaugusto suggested:

tl;dr: easier to keep track of what balance proof was used to close the channel, if the closing participant uses an old balance proof.

`settleChannel` needs the actual balance data. Participants can use old balance proofs to `closeChannel` and `updateNonClosingBalanceProof`, therefore a participant calling `settleChannel` needs to easily retrieve the old balance proof data used by his partner.

Currently, the `nonce` and `balance_hash` can be retrieved from the contract through `getChannelParticipantInfo`. 
However, it would be good to have the `nonce` of the balance proof in the actual events emitted on closing/updating:
- easier to process in the Raiden client
- easier to follow & debug the on-chain channel transactions in time (even after the channel data was removed)

`updateNonClosingBalanceProof` already has the `nonce` in the event, because it is needed by the Monitoring Service.
